src/VisualStudio/Core/Def/PublicAPI.Shipped.txt

```diff
@@ -11,5 +11,3 @@ static Microsoft.VisualStudio.LanguageServices.Progression.GraphNodeCreation.Cre
 static Microsoft.VisualStudio.LanguageServices.Progression.GraphNodeCreation.CreateNodeIdAsync(Microsoft.CodeAnalysis.ISymbol symbol, Microsoft.CodeAnalysis.Solution solution, System.Threading.CancellationToken cancellationToken) -> System.Threading.Tasks.Task<Microsoft.VisualStudio.GraphModel.GraphNodeId>
 static Microsoft.VisualStudio.LanguageServices.RQName.From(Microsoft.CodeAnalysis.ISymbol symbol) -> string
 virtual Microsoft.VisualStudio.LanguageServices.VisualStudioWorkspace.GetFilePath(Microsoft.CodeAnalysis.DocumentId documentId) -> string
-XamlGeneratedNamespace.GeneratedInternalTypeHelper
```

Comment:
This is terrifying this was ever included.

Reply:
Yeah :-|